src/worldmap.ts

```diff
@@ -140,6 +156,10 @@ export default class WorldMap {
   }
 
   updateCircles(data) {
+    const sizes = data.map(x => this.getValue(x));
+    this.lowestValue = Math.min(...sizes);
+    this.highestValue = Math.max(...sizes);
+    this.valueRange = this.highestValue - this.lowestValue;
```

Comment:
The exact same logic can be seen in the `createCircles` method.

Would it make sense to extract the logic into a method, which receives the `sizes` as a parameter?

Reply:
awkward ...  did it even better hehehe